service/controller/v14/version_bundle.go

```diff
@@ -47,6 +37,6 @@ func VersionBundle() versionbundle.Bundle {
 			},
 		},
 		Name:    "azure-operator",
-		Version: "2.9.0",
+		Version: "2.10.0",
```

Comment:
I figured `2.10.0` is the next version we want to have? No major k8s version upgrades or so atm. so for me this made sense.